style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.hover:hover {
+  cursor: pointer;
+}
+
+/* Styles for mobile devices with a minimum screen width of 375px */
+@media (min-width: 375px) {
+  .navbar {
+    padding: 14px 24px;
+  }
+
+  .main {
+    flex-direction: column;
+    min-height: 80vh;
+    row-gap: 12px;
+    background-image: url(img/header-shapes\ mobile.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 0 24px 24px;
+  }
```

Comment:
- [ ] Great work applying styles to your headline section 💯. However, please kindly make sure that you use exactly the same background provided in the Figma design, currently, there is one geometric pattern missing in your background and the left bottom border is not rounded🤓.

This is your design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/235034069-e078424c-e149-4db5-aff1-0b52d5e56c09.png)

This is the Figma design 👇🏽 (almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/235034087-1062bfd8-13fd-4fea-92d5-388ac27a0a48.png)